statemachine/contrib/diagram.py

```diff
@@ -9,7 +9,7 @@
 class DotGraphMachine:
     graph_rankdir = "LR"
     """
-    Diretion of the graph. Defaults to "LR" (option "TB" for top botton)
+    DireCtion of the graph. Defaults to "LR" (option "TB" for top botton)
```

Comment:
Still a typo that persists: `s/DireCtion/direction/g` :)

Reply:
Sorry - my spellcheck must have missed this - correcte in new commit.